iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/ConnectVoltageLevelOnLine.java

```diff
@@ -85,10 +56,9 @@ public ConnectVoltageLevelOnLine(String voltageLevelId, String bbsOrBusId, Strin
      *
      * NB: This constructor will eventually be package-private, please use {@link CreateLineOnLineBuilder} instead.
      */
-    public ConnectVoltageLevelOnLine(double percent, String voltageLevelId, String bbsOrBusId, String line1Id, String line1Name,
```

Comment:
you remove the "public" to force builder use ?

Reply:
Yes, since it is a breaking change anyway, I made the constructor package-private and deleted the deprecated ones.

Reply:
so can we rename percent in this pr ?

Reply:
Yes, we can!

Reply:
Does `position` sound OK? with javadoc for the unit ? Or `positionPercent` ?

Reply:
I prefer `positionPercent`